qiskit/providers/models/pulsedefaults.py

```diff
@@ -108,7 +106,7 @@ def __init__(self, name: str, qubits=None, sequence=None, **kwargs):
         Args:
             name (str): The name of the command
             qubits: The qubits for the command
-            sequence (PulseQobjInstruction): The sequence for the Command
+            sequence (List[PulseQobjInstruction]): The sequence for the Command
```

Comment:
I assume this was just a doc bug based on the argument name being sequence and not an api change. So this always took a list and the type hint was just wrong.

Reply:
Okey, I reverted the change in [8c82a02](https://github.com/Qiskit/qiskit-terra/pull/8885/commits/8c82a021c23a2002670d56c24f9d8e03e39db3e5)